syntax = "proto3";

package tutorial;

import "google/protobuf/any.proto";

message AnyDemo {
  google.protobuf.Any any_value = 2;
}